src/main/js/store/actions/session.js

```diff
@@ -77,7 +86,14 @@ export function verifyLogin() {
                 dispatch(loggedOut());
             } else {
                 const token = response.headers.get('X-CSRF-TOKEN');
-                dispatch(loggedIn({ csrfToken: token }));
+                response.json().then(body => {
+                    const { message, saml_enabled } = body;
+                    if (message === 'Authenticated') {
```

Comment:
We could check for a 204 here or something instead of looking for a String message body.

Reply:
I'll check the status code.